source/features/linkify-code.tsx

```diff
@@ -43,6 +43,7 @@ void features.add(import.meta.url, {
 	exclude: [
 		pageDetect.isGist,
 	],
+	deduplicate: 'has-rgh-inner',
```

Comment:
This fixes a similar bug with `linkify-code`, but since it runs on more pages than `restore-file` it might cause duplication bugs elsewhere.

Should we split the init and use `has-rgh-inner` only with `isPRFiles` or is that unnecessary?

Reply:
It's worth testing other pages. If this breaks the feature elsewhere it's not worth the change as is

Reply:
I dropped it for now, I think the fix there would be to move the deduplication logic out of the manager and into the feature itself. I think it'll just have to use `deduplication:false` since we already use `:not(linkifiedURLClass)`